be/src/Project/Web/Buyer/Buyer.Web/Areas/Products/Services/Products/ProductsService.cs

```diff
@@ -60,42 +82,48 @@ public async Task<PagedResults<IEnumerable<CatalogItemViewModel>>> GetProductsAs
         {
             var catalogItemList = new List<CatalogItemViewModel>();
             
-            var pagedProducts = await this.productsRepository.GetProductsAsync(ids, categoryId, sellerId, language, searchTerm, hasPrimaryProduct, pageIndex, itemsPerPage, token, nameof(Product.Name));
+            var pagedProducts = await _productsRepository.GetProductsAsync(ids, categoryId, sellerId, language, searchTerm, hasPrimaryProduct, pageIndex, itemsPerPage, token, nameof(Product.Name));
 
             if (pagedProducts?.Data != null)
             {
+                if (_options.Value.CompletionDatesUrl.IsNullOrEmpty() is false)
```

Comment:
suggestion: it should be: 

if (string.IsNullOrWhitespace(_options.Value.CompletionDatesUrl))